soccer/src/soccer/coach/coach_node.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <rj_msgs/msg/play_state.hpp>
+#include <rj_msgs/msg/coach.hpp>
+#include <rj_msgs/msg/global_override.hpp>
+
+#include <rj_geometry_msgs/msg/point.hpp>
+
+#include "game_state.hpp"
+
+enum match_situation {
```

Comment:
I think it wants camel case 
```suggestion
enum MatchSituation {
```